src/app/components/pages/SetAssignments.tsx

```diff
@@ -113,6 +113,14 @@ const AssignGroup = ({groups, board, assignBoard}: BoardProps) => {
     const yearRange = range(currentYear, currentYear + 5);
     const currentMonth = (new Date()).getMonth() + 1;
 
+    function setScheduledStartDateAtSevenAM(e: ChangeEvent<HTMLInputElement>) {
+        const utcDate = e.target.valueAsDate as Date;
+        const sevenAM = 7 * 60 * 60 * 1000;
+        const scheduledDate = new Date(utcDate.getTime() + sevenAM);
```

Comment:
I don't think this is correct? For me the timestamp this creates is always 8am BST, because the `utcDate` is midnight UTC (so _1am_ BST) and then adding 7 hours leads to 8am BST. I have tested this suggestion (whilst trying to work out why the change never reached the API).
```suggestion
        const scheduledDate = new Date(utcDate.getFullYear(), utcDate.getMonth(), utcDate.getDate(), 7);
```

Reply:
Much more readable, thanks!